runtime/model-protobuf/src/test/java/io/aklivity/zilla/runtime/model/protobuf/internal/ProtobufModelTest.java

```diff
@@ -38,60 +39,99 @@
+                                                        required string required_field = 1;
+                                                        optional string optional_field = 2;
+                                                        optional int32 optional_with_default = 3 [default = 42];
+                                                        repeated string repeated_field = 4;
+                                                        optional bool bool_field = 5 [default = true];
+                                                        enum Status {
+                                                            UNKNOWN = 0;
+                                                            ACTIVE = 1;
+                                                            INACTIVE = 2;
+                                                        }
+                                                        optional Status status = 6 [default = ACTIVE];
+                                                        message NestedMessage {
+                                                            required int32 nested_id = 1;
+                                                            optional string nested_name = 2;
+                                                        }
+                                                        optional NestedMessage nested = 7;
+                                                        repeated NestedMessage nested_list = 8;
+                                                        oneof choice {
+                                                            string choice_string = 9;
+                                                            int32 choice_int = 10;
+                                                        }
+                                                        map<string, int32> string_int_map = 11;
+                                                        extensions 100 to 199;
+                                                    }
+                                                    extend Proto2Message {
+                                                        optional string extension_field = 100;
+                                                    }
+                                                    message ComplexProto2Message {
+                                                        required Proto2Message main_message = 1;
+                                                        optional bytes binary_data = 2;
+                                                        repeated double double_values = 3 [packed = true];
+                                                        optional group GroupField = 4 {
+                                                            optional int32 group_id = 5;
+                                                            optional string group_name = 6;
+                                                        }
+                                                    }""";
```

Comment:
No need to indent so far, the `"""` takes care of ignoring the initial whitespace on each line, even if not aligned with the location of `"""` on the line above.